__tests__/app.test.js

```diff
@@ -61,3 +62,38 @@ describe.only('GET /api', () => {
             });
     })
 })
+
+describe.only('GET /api/articles/:article_id', () => {
```

Comment:
Don't forget to remove .only's before PRs to make sure all your tests are still passing 🧠